### ISSUE TITLE
Can't run tests

### ISSUE THREAD

**Issue by dangreen** [lucasmotta/sass-module-importer#19]
```
sass-module-importer(master*)$ npm test

> sass-module-importer@1.2.1 pretest /Users/dangreen/github/sass-module-importer
> npm run lint && npm run build


> sass-module-importer@1.2.1 lint /Users/dangreen/github/sass-module-importer
> eslint src/index.js test.js


> sass-module-importer@1.2.1 build /Users/dangreen/github/sass-module-importer
> rollup -c

Path must be a string. Received undefined
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.resolve (path.js:1148:7)
    at resolveId$1 (/Users/dangreen/github/sass-module-importer/node_modules/rollup-plugin-node-resolve/dist/rollup-plugin-node-resolve.cjs.js:44:15)
    at /Users/dangreen/github/sass-module-importer/node_modules/rollup/www/ROLLUP/rollup/src/utils/first.js:10:31
    at tryCatch (/Users/dangreen/github/sass-module-importer/node_modules/rollup/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:185:12)
    at invokeCallback (/Users/dangreen/github/sass-module-importer/node_modules/rollup/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:197:13)
    at publish (/Users/dangreen/github/sass-module-importer/node_modules/rollup/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:168:7)
    at flush (/Users/dangreen/github/sass-module-importer/node_modules/rollup/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/asap.js:87:5)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki
```

**Comment by dangreen** [comment]
Fixed in https://github.com/lucasmotta/sass-module-importer/pull/20

**Comment by lucasmotta** [comment]
Fixed on 9870e5c76807f10d3ea4bd3d6b351211c771dab9
